setup.cfg

```diff
@@ -43,7 +43,23 @@ max-line-length = 100
 format = pylint
 statistics = 1
 show-source = 1
-exclude = build,.git,.circleci,.coverage,.eggs,.gitignore,.idea,coverage,dist,*.egg*,venv,venv_py3,.tox,__pycache__
+exclude =
+    build,
+    spotify_tensorflow/tensorflow_transform,
```

Comment:
I wonder whether it's safe to call this directory `tensorflow_transform`, and that at some point there wouldn't be not misunderstanding or shadowing happening - what do you think?

Reply:
I think this is a temp hack - it's good to make it obvious it's copied code. Hopefully it'll get nuked soon. Also there is no scope shadowing risk since you can import both `spotify_tensorflow.tensorflow_transform.foo` and `tensorflow_transform.foo`